docs/integrations/source_version_control.md

```diff
@@ -14,19 +14,35 @@ repos:
         # supported by your project here, or alternatively use
         # pre-commit's default_language_version, see
         # https://pre-commit.com/#top_level-default_language_version
-        language_version: python3.9
+        language_version: python3.11
 ```
 
 Feel free to switch out the `rev` value to something else, like another
 [tag/version][black-tags] or even a specific commit. Although we discourage the use of
 branches or other mutable refs since the hook [won't auto update as you may
 expect][pre-commit-mutable-rev].
 
-If you want support for Jupyter Notebooks as well, then replace `id: black` with
-`id: black-jupyter`.
+## Jupyter Notebooks
+
+If you want to include support for Jupyter Notebooks, then simply replace the hook's
+`id: black` with `id: black-jupyter` in the `.pre-commit-config.yaml`:
+
+```yaml
+repos:
+  - repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black-jupyter
+        description: Black, with Jupyter Notebook support
```

Comment:
[The `description` key is only used in `.pre-commit-hooks.yaml`, not `.pre-commit-config.yaml`](https://pre-commit.com/#pre-commit-configyaml---hooks). In other words, it goes on the provider side, not the consumer side.

Reply:
Good point, I removed the `description` field.

One question I had in my head when I made the original issue was, does `black-jupyter` run _only_ on Jupyter Notebooks?  And if yes, should I still keep the `id: black` line in my `.pre-commit-config.yaml`?  The answer is no, `black-jupyter` runs on both normal Python files and Jupyter Notebooks.

I thought having the `description` entry would answer this question within the `pre-commit` file, instead of requiring devs to pull up the `black` docs to search.

I think once one already knows `black-jupyter` runs on normal Python files, this `description` field isn't necessary.  It's only useful when one doesn't have this knowledge.  Thus, I think you're right let's remove the `description`, and hopefully new users find it intuitive (and can read the updated docs otherwise).

---

_Aside_

I would most like to see the below, like one does for `flake8` configs.  I think stating `jupyter` under `additional_dependencies` is intuitive.  However, it doesn't quite work as it's missing `types_or` for `jupyter` file type:

```yaml
repos:
  - repo: https://github.com/psf/black
    rev: 23.3.0
    hooks:
      - id: black
        additional_dependencies:
          - jupyter  # Also run on Jupyter Notebooks
```